examples/inverse/plot_mne_point_spread_function.py

```diff
@@ -62,15 +62,17 @@
 # stc_psf_eegmeg.save('psf_eegmeg')
 # stc_psf_meg.save('psf_meg')
 
-time_label = "EEGMEG %d"
-brain_eegmeg = stc_psf_eegmeg.plot(hemi='rh', subjects_dir=subjects_dir,
-                                   time_label=time_label,
-                                   figure=create_3d_figure(size=(500, 500)))
+with use_3d_backend("mayavi"):
+    time_label = "EEGMEG %d"
+    brain_eegmeg = stc_psf_eegmeg.plot(hemi='rh', subjects_dir=subjects_dir,
+                                       time_label=time_label,
+                                       figure=create_3d_figure(
+                                           size=(500, 500)))
 
-time_label = "MEG %d"
-brain_meg = stc_psf_meg.plot(hemi='rh', subjects_dir=subjects_dir,
-                             time_label=time_label,
-                             figure=create_3d_figure(size=(500, 500)))
+    time_label = "MEG %d"
+    brain_meg = stc_psf_meg.plot(hemi='rh', subjects_dir=subjects_dir,
+                                 time_label=time_label,
+                                 figure=create_3d_figure(size=(500, 500)))
```

Comment:
I don't understand why you need to use `use_3d_backend` in these examples. `stc.plot` should use PySurfer which will use mayavi anyway

Reply:
It's for the `create_3d_figure()` function which relies on the `_Renderer` now. Before the port, it was `mlab.figure()`.

Reply:
You don't need `create_3d_figure`, use `stc.plot(..., size=(500, 500))` instead

Reply:
Oh wait, that could lead to a problem with it plotting into the same Mayavi figure because it's the same subject name.

In that case, perhaps do `figure=1` in the first plot and `figure=2` in the second one, and it should spawn separate plots for you.

Reply:
(one of these two solutions should hopefully work and allow you to avoid the MNE context manager business)